internal/mode/static/nginx/file/folders_test.go

```diff
@@ -14,15 +14,14 @@ import (
 
 func writeFile(t *testing.T, name string, data []byte) {
 	t.Helper()
+	g := NewWithT(t)
 
 	//nolint:gosec // the file permission is ok for unit testing
-	if err := os.WriteFile(name, data, 0o644); err != nil {
-		t.Fatal(err)
-	}
+	g.Expect(os.WriteFile(name, data, 0o644)).Should(Succeed())
```

Comment:
```suggestion
	g.Expect(os.WriteFile(name, data, 0o644)).To(Succeed())
```

I think `To` makes more sense from a readability standpoint. It does the same thing.

Reply:
I did a quick search and it looks like we're pretty split even on using `Should(Succeed())` and `To(Succeed())`, do you think it would be a good idea to change the rest of the `Should(Succeed())`'s to `To(Succeed())`

Reply:
I'd be curious what the rest of the team thinks. It's mainly about what sound the best when read like a normal sentence. If it could go either way, then it doesn't really matter.

Reply:
I also prefer `To` but since they're interchangeable I don't think we need to replace all the `Should`s with `To`s

Reply:
Alright, I made the change above and I'll leave it be